lighthouse/classes/event_properties/validations.py

```diff
@@ -0,0 +1,85 @@
+from lighthouse.classes.event_properties.interfaces import EventPropertyAbstract  # type: ignore
```

Comment:
We shouldn't need to ignore any of our own modules. It could be that the folder is missing the `__init__.py` which mypy seems to need to resolve types.

Reply:
Yes, that solved the issue; I didn't know what the __init__.py files were for